fdbserver/masterserver.actor.cpp

```diff
@@ -1922,13 +1940,32 @@ ACTOR Future<Void> masterCore(Reference<MasterData> self) {
 	} else {
 		// Recruit and seed initial shard servers
 		// This transaction must be the very first one in the database (version 1)
-		for (const auto& s : self->configuration.splits) {
-			TraceEvent("InitialSplitPoint", self->dbgid).detail("Point", s.toString());
-		}
 		seedShardServers(recoveryCommitRequest.arena, tr, seedServers);
```

Comment:
I was thinking adding key ranges defined by split points to this call so that storage servers can be initialize with its ranges.